From 1f656fe7bff0e26de07a5811bf69b5c872a2941e Mon Sep 17 00:00:00 2001
From: Keerthy <j-keerthy@ti.com>
Date: Thu, 5 Sep 2019 13:50:30 +0530
Subject: [PATCH 7/8] configs: j721e_evm_a72_defconfig: Add support for pm2 som
 with tps65941 PMIC

Add support for pm2 som with tps65941 PMIC. This enables
dynamic matching of dtb based on the EEPROM per SoM.

Signed-off-by: Keerthy <j-keerthy@ti.com>
Signed-off-by: Nikhil Devshatwar <nikhil.nd@ti.com>
---
 configs/j721e_evm_a72_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/j721e_evm_a72_defconfig b/configs/j721e_evm_a72_defconfig
index e71cdc2..c9eda97 100644
--- a/configs/j721e_evm_a72_defconfig
+++ b/configs/j721e_evm_a72_defconfig
@@ -60,6 +60,7 @@ CONFIG_CMD_UBI=y
 CONFIG_OF_CONTROL=y
 CONFIG_SPL_OF_CONTROL=y
 CONFIG_DEFAULT_DEVICE_TREE="k3-j721e-common-proc-board"
+CONFIG_OF_LIST="k3-j721e-common-proc-board k3-j721e-tps65917-proc-board"
 CONFIG_SPL_MULTI_DTB_FIT=y
 CONFIG_SPL_MULTI_DTB_FIT_NO_COMPRESSION=y
 CONFIG_ENV_IS_IN_MMC=y
-- 
2.7.4

